src/coreclr/src/jit/codegencommon.cpp

```diff
@@ -6118,14 +6117,16 @@ regNumber CodeGen::genGetZeroReg(regNumber initReg, bool* pInitRegModified)
 // genZeroInitFrame: Zero any untracked pointer locals and/or initialize memory for locspace
 //
 // Arguments:
-//    untrLclHi        - (Untracked locals High-Offset)  The upper bound offset at which the zero init
-//                                                       code will end initializing memory (not inclusive).
-//    untrLclLo        - (Untracked locals Low-Offset)   The lower bound at which the zero init code will
-//                                                       start zero initializing memory.
-//    initReg          - A scratch register (that gets set to zero on some platforms).
-//    pInitRegModified - Sets a flag that tells the callee whether or not the initReg register got zeroed.
+//    untrLclHi      - (Untracked locals High-Offset)  The upper bound offset at which the zero init
+//                                                     code will end initializing memory (not inclusive).
+//    untrLclLo      - (Untracked locals Low-Offset)   The lower bound at which the zero init code will
+//                                                     start zero initializing memory.
+//    initReg        - A scratch register (that gets set to zero on some platforms).
+//    pInitRegZeroed - Sets a flag that tells the callee whether or not the initReg register got zeroed.
```

Comment:
This might be more clear as:
```
//    pInitRegZeroed - OUT parameter. *pInitRegZeroed is set to 'true' if this method sets initReg register to zero,
//                     'false' if initReg was set to a non-zero value, and left unchanged if initReg was not touched.
```

Reply:
Fixed